ignored-tests.json

```diff
@@ -0,0 +1,180 @@
+[
+  "Accessibility autocomplete",
+  "Accessibility filtering children of leaf nodes checkbox with and tabIndex and label should not have children",
+  "Accessibility filtering children of leaf nodes checkbox without label should not have children",
+  "Accessibility filtering children of leaf nodes non editable textbox with role and tabIndex and label should not have children",
+  "Accessibility filtering children of leaf nodes plaintext contenteditable plain text field with role should not have children",
+  "Accessibility filtering children of leaf nodes rich text editable fields should have children",
+  "Accessibility filtering children of leaf nodes rich text editable fields with role should have children",
+  "Accessibility filtering children of leaf nodes root option should return null when the element is no longer in DOM",
+  "Accessibility filtering children of leaf nodes root option should support the interestingOnly option",
+  "Accessibility filtering children of leaf nodes root option should work a button",
+  "Accessibility filtering children of leaf nodes root option should work a menu",
+  "Accessibility filtering children of leaf nodes root option should work an input",
+  "Accessibility filtering children of leaf nodes should not report text nodes inside controls",
+  "Accessibility get snapshots while the tree is re-calculated",
+  "Accessibility keyshortcuts",
+  "Accessibility multiselectable",
+  "Accessibility orientation",
+  "Accessibility roledescription",
+  "Accessibility should report uninteresting nodes",
+  "Accessibility should work",
+  "Autofill ElementHandle.autofill should fill out a credit card",
+  "Browser specs Browser.isConnected should set the browser connected state",
+  "Browser specs Browser.userAgent should include WebKit",
```

Comment:
This is just getting the default one, no override, should this be part of capabilities? I think we need it.

Reply:
Yes, but it definitely should not include WebKit

Reply:
The test does not test for that though... It does branch to `WebKit` and `Gecko`

Reply:
Good point, we should rename the test!

Reply:
https://github.com/puppeteer/puppeteer/pull/11699